app/views/homepage/index.html.erb

```diff
@@ -4,14 +4,17 @@
       <%= render partial: 'shared/header', locals: { isCover: true } %>
       <div class="c-hero">
         <h1 class="title -light -huge">Advancing financial inclusion</h1>
-        <p class="subtitle">A platform for <a href="#">governments</a>,
-          <a href="#">researchers</a> and <a href="#">FSP Techs.</a>
+        <p class="subtitle">A platform for
+          <a class="js-anchor" data-anchor="#governments" href="#">governments</a>,
```

Comment:
Put the content of `data-anchor` in the attribute `href`. In addition to removing code, it makes the code works as a progressive enhancement: if the JS doesn't load, the user will still be scrolled to the relevant sections.